LV-759 Hybrisa Prospera port [IDB IGNORE] [TM only]

AlanParkerMasters opened the pull request:
# About the pull request

![hybrisa8](https://github.com/user-attachments/assets/99612044-0277-4615-a00b-137a2992feb8)


Adds LV-759, Hybrisa Prospera to PVE.
Also adds a 'repaired' version for HAI, HvH, greenshifts/blueshifts, whatever. See screenshots for details.
This took a few days to complete. 
Much love to everyone that helped put this together. Mad love to the people who made this map to begin with.

Expect bugs and issues here and there, I'll be ironing most of this out in a few days.
IMPORTANT: There is quite a lot of icon organizational issues I want to deal with in the coming days so just give me some time.


# Explain why it's good for the game
These streets are cold and windy.


# Testing Photographs and Procedure
All of the wonderful residents of Hybrisa
![hybrisa-civs](https://github.com/user-attachments/assets/2ba8d114-1369-41ac-9998-90f065e3e960)


<details>
<summary>More screenshots of Hybrisa</summary>

![hybrisa1](https://github.com/user-attachments/assets/24b5448b-ccab-44ec-943d-36e53f3b2f5f)

![hybrisa2](https://github.com/user-attachments/assets/0c3f4a8d-748e-436e-a37f-f68c89f2cbb9)

![hybrisa3](https://github.com/user-attachments/assets/0258eb65-fe1f-45fd-88c7-477efe5d2922)

![hybrisa4](https://github.com/user-attachments/assets/250c9c7b-69b4-4782-a6f6-e0f6aed0b491)

![hybrisa5](https://github.com/user-attachments/assets/32fbd84d-8506-4b3b-ba8c-37106cb36ce4)

![hybrisa6](https://github.com/user-attachments/assets/bb9a9e34-5e5a-46e0-a15c-7b655ee5fda9)

![hybrisa7](https://github.com/user-attachments/assets/eca11045-051e-4827-a118-1688a508a81b)

![hybrisa9](https://github.com/user-attachments/assets/213f04a2-8f2b-4569-aaa5-8d7ad606d0a1)

![hybrisa10](https://github.com/user-attachments/assets/ef4edf86-6217-486b-8197-1bd03ac9a1bf)

</details>

<details>
<summary>"Repaired" Hybrisa</summary>

![hcc1](https://github.com/user-attachments/assets/50a75fe7-d73b-444d-a993-b009371b5311)

![hcc2](https://github.com/user-attachments/assets/ab9c9a3c-901d-46ac-aabd-5232ba14b29c)

![hcc3](https://github.com/user-attachments/assets/70ea543a-e4c4-4cfe-aa4e-189f5a81aeab)

![hcc4](https://github.com/user-attachments/assets/7071177e-8ace-4bd0-8159-1d51c0d4efa3)

![hcc5](https://github.com/user-attachments/assets/86135c82-7c67-4bb9-8e80-729e2ec2405f)

![hc1](https://github.com/user-attachments/assets/cb73ae87-69f7-41b0-aba4-fc6bd257053a)

![hc2](https://github.com/user-attachments/assets/2da86ece-61a2-49bb-862f-f66468f96f01)

![hc3](https://github.com/user-attachments/assets/023b58f2-3a60-4cb5-8b16-cdfcac96dc5f)

![hc4](https://github.com/user-attachments/assets/570ebd4b-09e7-41be-9584-74bcd7014b0a)

![hc5](https://github.com/user-attachments/assets/1e74c5ed-3eb5-4f71-bd54-cfb18e3219b9)

![hc6](https://github.com/user-attachments/assets/0b63bc28-68c4-4f1a-98c1-a6201c7ccb7c)

![hc7](https://github.com/user-attachments/assets/83efb283-1b0e-4731-a345-73adcc484448)

![hc8](https://github.com/user-attachments/assets/7c39169c-ff22-44af-9521-04f440fe4395)

![hc9](https://github.com/user-attachments/assets/7d689b43-74b2-47ba-92c0-af7a65240aa7)

![hc10](https://github.com/user-attachments/assets/742b58d6-ecff-40d8-9ba6-f8f2d0096551)

</details>

# Changelog
:cl:
add: LV-759 Hybrisa Prospera
add: LV-759 Hybrisa Prospera repaired
add: a LOT of assets
/:cl:

AlanParkerMasters committed:
aa

AlanParkerMasters committed:
aa

AlanParkerMasters committed:
yup

AlanParkerMasters committed:
finally

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
det hat fixes/fedora

AlanParkerMasters committed:
ANOTHER hat fix

AlanParkerMasters committed:
even more hat fixes (guh)

Drulikar commented:
IDB ignored until final commit.

AlanParkerMasters committed:
line art and fixes

AlanParkerMasters committed:
fixes (corpse spawner bug)

AlanParkerMasters committed:
corpse spawner removal bug

AlanParkerMasters committed:
more corpse spawner fixes

AlanParkerMasters committed:
adds two autodocs to Hybrisa hospital for PVP map

AlanParkerMasters committed:
Hybrisa repaired map complete

AlanParkerMasters committed:
hybrisa repaired map fix

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
fixes planes issue (repaired Hybrisa)

AlanParkerMasters committed:
Hybrisa repaired map fix

AlanParkerMasters committed:
var edit fix (repaired)

AlanParkerMasters committed:
couple of varedited .dmi's removed

AlanParkerMasters committed:
proper spacecraft fuel tanks (repaired-hy)

AlanParkerMasters committed:
atm fixes and giant driller addition (clean-hy)

AlanParkerMasters committed:
more fixes

AlanParkerMasters committed:
banned neighbor tiles fixes (clean-hy)

AlanParkerMasters committed:
more banned neighbor fixes

AlanParkerMasters committed:
another neighbor fix

AlanParkerMasters committed:
even more neighbor fixes

AlanParkerMasters committed:
a

AlanParkerMasters committed:
aa

AlanParkerMasters committed:
dmi fix (c-h)

AlanParkerMasters committed:
twe airbase clean Hybrisa fix

AlanParkerMasters committed:
removes weeds from repaired

AlanParkerMasters committed:
replaces Engi ship with expanded caverns (repaired-hy)

AlanParkerMasters committed:
replaces some unacid/explode proof shutters (rep-hy)

AlanParkerMasters committed:
more fixes (rep-hy)

AlanParkerMasters committed:
Adds and fixes proper phone lines (rep-hy)

AlanParkerMasters committed:
proper functioning ID doors + klaxon blaring removal

AlanParkerMasters committed:
check fix

AlanParkerMasters committed:
anchors and layers Hybrisa decals properly

AlanParkerMasters committed:
another decal fix

AlanParkerMasters committed:
makes Hybrisa dirt and trash decals cleanable

AlanParkerMasters committed:
Merge branch 'master' into hybrisa_port

AlanParkerMasters committed:
actually gives IDs accountant numbers

AlanParkerMasters committed:
test fix

AlanParkerMasters committed:
door ID fix

AlanParkerMasters committed:
area check fix

AlanParkerMasters committed:
adds working ambulances and cop cars

AlanParkerMasters committed:
aa

AlanParkerMasters committed:
random map fixes (clean hy)

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
bed.dm conflict fix

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
corporate rifle back

AlanParkerMasters committed:
brings back pmc beret

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
removes random survivor weaponry by request

AlanParkerMasters committed:
check fixes for dmi file limits

workarounds for dmi file limit in uniform0.dmi by moving most Hybrisa uniforms to uniform1.dmi until icon reorganization

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
<!-- tgs_test_merge_comment -->
## Test merge deployment history:

<details><summary>Test Merge Deployed @ 03/23/2025 16:10:37 +00:00:</summary>

##### Server Instance
PVECM: Red

##### Merged By
MessageCypher

##### Revision
Origin: 1172d1f1588b01be620a11e22ea32db183fc2560
Pull Request: 3f33f66e194c0587be7e5565138069eaa3333b79
Server: c51c4e24a196331da80f8ceaff89c8849b3205a0
</details>

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
vehicle name fix, shutter dupe fix, container layer fix

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
fix random obj preset

AlanParkerMasters committed:
MRE and tear gas fixes

AlanParkerMasters committed:
wy mre fixes

AlanParkerMasters committed:
sg armor fix

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
Merge branch 'master' into hybrisa_port

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Zonespace27 commented on code/modules/mob/living/carbon/human/ai/defense_creator.dm:
```diff
@@ -205,12 +205,6 @@
 	icon_state = "wy_sentry_heavy"
 	path_to_spawn = /obj/structure/machinery/defenses/sentry/dmr/wy
 
-/datum/human_ai_defense/defense/sentry/wy/static_gun
```
Why?

AlanParkerMasters replied:
Not sure, this was removed while I was porting a bunch of stuff, I believe mainstream got rid of it
Re-added

Zonespace27 reviewed (changes requested):
HAI

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
readds static HAI sentry

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
sentry fix

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
ammo casings fix

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

AlanParkerMasters committed:
Merge branch 'master' of https://github.com/AlanParkerMasters/cmss13-pve into hybrisa_port

BonniePandora committed:
Merge branch 'master' into pr/803

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

BonniePandora committed:
area order fix

BonniePandora committed:
Icon shuffling

BonniePandora committed:
Reeeee

BonniePandora committed:
a

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

BonniePandora committed:
Merge branch 'master' into pr/803

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

BonniePandora committed:
Merge branch 'master' into pr/803

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

BonniePandora committed:
Merge branch 'master' into pr/803

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

BonniePandora commented:
Reopened to resolve conflicts for GM usage during the PR freeze

BonniePandora committed:
Merge branch 'master' into pr/803

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

Tristan63 commented:
conflict 
<img width="778" height="306" alt="image" src="https://github.com/user-attachments/assets/e61d1563-fe49-47f6-86ab-485d8f3dbcd9" />
